Core.DomainServices/SystemUsage/BuildItSystemUsageOverviewReadModelOnChangesHandler.cs

```diff
@@ -93,7 +94,12 @@ public void Handle(EntityUpdatedEvent<BusinessType> domainEvent)
             _pendingReadModelUpdateRepository.Add(PendingReadModelUpdate.Create(domainEvent.Entity.Id, PendingReadModelUpdateSourceCategory.ItSystemUsage_BusinessType));
         }
 
-        public void Handle(EntityUpdatedEvent<LocalBusinessType> domainEvent)
+        public void Handle(EntityCreatedEvent<LocalBusinessType> domainEvent)
+        {
+            //Point to parent id since that's what the readmodel knows about
+            _pendingReadModelUpdateRepository.Add(PendingReadModelUpdate.Create(domainEvent.Entity.OptionId, PendingReadModelUpdateSourceCategory.ItSystemUsage_BusinessType));
```

Comment:
ja i princippet kan den jo have været slået fra (udløbet) og slået til igen uden at indstillingen har ændret sig så for local type er det ekstra kompliceret

tror vi skal have lavet en løsning til at generalisere det på et tidspunkt men indtil videre må vi leve videre efter denne viden